resources/default-settings/jukebox.default.yaml

```diff
@@ -18,6 +18,7 @@ modules:
     host: hostif.linux
     bluetooth_audio_buttons: controls.bluetooth_audio_buttons
     gpio: gpio.gpioz.plugin
+    sync_shared: synchronisation.sync_shared
```

Comment:
The idea of the feature was to sync elements in the "shared" folder.
I took the same name as in v2 (here "audiofolders" and "shortcuts" had been the only elements in shared), to be identified as the same feature.

I didn't choose "remote" or "from remote" in the first place, as it doesn't say what is synced and the direction may not always be only to the phoniebox (further additions to sync files back to the server are possible, yet not currently planned).

But we can try to find a name that fits it all.

Reply:
I thought about it.. why don't you just call it `filesync` .. and the file structure could be like this

`filesync/remote` instead of `synchronisation/sync_shared`

in your `jukebox.yaml` you could register it this way

```yaml
  filesync: filesync.remote
```

Reply:
changed name to "synchronisation/rfidcards" ("sync_rfidcards" in jukebox.yaml)

"synchronisation" is the generic component, "rfidcards" the specific implementation. 
For now RFID card data (audiofolders and card database entries) are the only synced items.